airflow/providers/google/cloud/transfers/gcs_to_bigquery.py

```diff
@@ -286,7 +286,7 @@ def execute(self, context):
         else:
             schema_fields = self.schema_fields
 
-        source_uris = [f'gs://{self.bucket}/{source_object}' for source_object in self.source_objects]
+        source_uris = [f'gs://{self.bucket}/{source_object}' for source_object in self.source_objects] if type(self.source_objects) is list else [f'gs://{self.bucket}/{self.source_objects}'] 
```

Comment:
Wouldn't it be better to convert the string to list and then the code will work just fine without a special case for string?

Reply:
But if the input is a list, wouldn't it be converted to a 2-dimensional list?
```
self.source_objects = "file1.csv"
source_object = [self.source_objects]
output: ["file1.csv"]

self.source_objects = ["file1.csv", "file2.csv", "file3.csv"]
source_object = [self.source_objects]
output: [["file1.csv", "file2.csv", "file3.csv"]]
```

Reply:
No. I was thinking of converting the type to List (if needed) so the logic part of the code won't need any modification as it works fine with List type. something like:
```
if isinstance(obj, str):
   obj=[obj]
```

Reply:
ahh, I was thinking about that solution too. So, it doesn't have to be one line code?